src/cgroups/v1/controller_type.rs

```diff
@@ -13,6 +15,8 @@ pub enum ControllerType {
 impl ToString for ControllerType {
     fn to_string(&self) -> String {
         match self {
+            Self::Cpu => "cpu".into(),
```

Comment:
Confusingly some systems have the CPU subsystem and CPU accounting subsystem mounted at the same path, or what's often called comounting. I may be missing something, but I don't believe we account for that. See the section **Mounting v1 Controllers** here: https://man7.org/linux/man-pages/man7/cgroups.7.html

Reply:
I check for this in the cgroup manager when we are constructing the subsystem paths.

Reply:
I totally missed that somehow. Looks good.